docs/modules/knowledge-graph/pages/index.adoc

```diff
@@ -0,0 +1,44 @@
+= Graph-based Knowledge Extraction and Traversal
+
+RAGStack offers two libraries supporting knowledge graph extraction and traversal, `ragstack-ai-knowledge-graph` and `ragstack-ai-knowledge-store`.
+
+A knowledge graph represents distinct entities (real world objects, like "Marie Curie" or "maize") as **nodes**. Nodes are connected by **edges** indicating relationships between them. Each edge includes the source (for example, "Marie Curie" the person), the target ("Nobel Prize" the award) and a type, indicating how the source relates to the target (for example, “won”).
+
+A graph database isn't required to use the knowledge graph libraries - RAGStack uses Astra DB or Apache Cassandra to store and retrieve graphs.
+
+*TL;DR:*
```

Comment:
The TL;DR seems longer than the original? Feels like this is less of a summary and rather more information?